ajax/libs/colofilter.css/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "colofilter.css",
+  "filename": "colofilter.min.css",
+  "version": "1.0.0",
+  "description": "Duotone filters made with CSS",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LukyVj/colofilter.css"
+  },
+  "homepage": "https://lukyvj.github.io/colofilter.css/",
+  "keywords": [
+    "colofilter",
+    "duotone",
+    "css"
+  ],
+  "author": "Lucas Bonomi",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/LukyVj/colofilter.css.git",
+    "basePath": "styles",
```

Comment:
Here is the file https://raw.githubusercontent.com/LukyVj/colofilter.css/master/styles/colofilter.css